pkgs/applications/audio/quodlibet/default.nix

```diff
@@ -46,7 +45,7 @@
   xvfb-run,
 }:
 
-python3.pkgs.buildPythonApplication rec {
+python3.pkgs.buildPythonApplication {
   pname = "quodlibet${tag}";
   version = "4.6.0";
```

Comment:
Probably lost the version update to "unstable-<date>" in the rebase?

Reply:
yeah, nice catch!